### ISSUE TITLE
All input queries should work without any commas

### ISSUE THREAD

**Issue by riordan** [pelias/pelias#189]
#tl;dr
All our input parameters need to work without expecting any commas (or punctuation)

# The Long Version
Users don't use commas when typing addresses on cell phones.

As @souperneon pointed out, iOS's default keyboard doesn't display commas
![from skitch](https://cloud.githubusercontent.com/assets/227472/11666086/a4e2a7ac-9db8-11e5-8ace-74105dca818a.png)

(Mainline Android's default keyboard + Samsung Keyboard are pro comma, but that doesn't mean our users are)

A quick paroozal of our logs for search showed a substantial number of searches lacking commas where we often expect them (logs are not being posted here for user privacy). This is true even of well-formed addresses that appear to being batch geocoded, but where our users are concatenating several fields into a single line (e.g. `182 E Island Lake Dr Roanoke, VA`) so we can parse it in the first place.

As such, we've got to get used to inputs without any punctuation for both `/search` and `/autocomplete`.

**Comment by stephenkhess** [comment]
:+1:

**Comment by souperneon** [comment]
++1 
1. punctuation is an extra character
2. sometimes not easily accessible as a character
3. more time
4. we have been trained by other search services (not naming names) to just predict what we are thinking, which has made us as users lazy. 

Commas and other punctuation is a nice to have but shouldn't mess up the quality of the results

**Comment by riordan** [comment]
This should be improved by Libpostal

**Comment by orangejulius** [comment]
Connects https://github.com/pelias/pelias/issues/152

Once that's closed, look for an update here, probably with good news.

**Comment by trescube** [comment]
Being tracked in pelias/pelias#152
